index.html

```diff
@@ -0,0 +1,119 @@
+    <!-- Bootstrap Icons CDN -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css" />
+    <link rel="stylesheet" href="style.css">
+    <title>Capstone 1</title>
+</head>
+
+<body>
+    <nav>
+        <div class="social">
+            <ul class="d-flex justify-content-end">
+                <li><a href="#"><img src="img\facebook.svg" alt="logo facebook"></a></li>
+                <li><a href="#"><img src="img\twitter.svg" alt="logo twitter"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li> <a href="#">Logout</a></li>
+            </ul>
+        </div>
+    </nav>
+    <div class="container-fluid m-0">
+        <header class="py-2">
+            <nav><img class="h-30 ps-3 mobilemenu" src="img/hamburguer.svg" alt="hamburguer menu">
+
+                <div class="desktopmenu d-flex justify-content-between">
+                    <a class="ms-5 ps-5 logo" href=""><img src="img/logo.png" alt="logo creative commons"></a>
+                    <ul>
+                        <a style="color: rgb(255, 51, 0);" href="about.html">
+                            <li>About</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Program</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Join</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5">Sponsor</li>
+                        </a>
+                        <a href="#">
+                            <li class="ms-5 me-3">News</li>
+                        </a>
+                        <a style="border: 3px solid rgb(255, 51, 0); border-radius: 5px;color: rgb(255, 51, 0);"
+                            href="#">
+                            <li class="mx-5">Campaign</li>
+                        </a>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+
+        <div class="container text-dark sm-fs-4">
+            <h2 class="pt-3 pb-3 mt-1" style="color: rgb(255, 51, 0);">"Hello, Sharing world"</h2>
+            <h1 style="color: rgb(255, 51, 0);">CONTAINER FLUID <br><br><br> ARTS FESTIVAL<br><br><br> 2022</h1><br>
+            <p class="bg-light p-1">Container Festival is without doubt the Bucaramanga's most famous music and
+                performing arts festival, if not the
+                world's. Embracing every genre across the spectrum </p>
+            <span class="date">2022.11.03(THU) ~ 06(SAT)</span><br>
+            <small>@National Museum, Modern Art Museum and more</small>
+        </div>
+    </div>
+
+    <div class="programwrapper">
+        <h1 class="text-center fs-3 " style="color: #fff;">Main Program</h1>
+        <hr class='mb-4'
```

Comment:
- [OPTIONAL] I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).

> NOTE: You've used `<h1>` tag once in this page, here `<h1>CONTAINER FLUID ...</h1>`, and that's great because this is the main title 💯

> NOTE: **Only Use H1 Elements Once Per Page for optimal SEO**